pkg/sdn/plugin/subnets.go

```diff
@@ -155,6 +165,37 @@ func (master *OsdnMaster) watchNodes() {
 	}
 }
 
+// Because openshift-sdn uses an overlay and doesn't need GCE Routes,
+// we need to clear the kubelet-added NetworkUnavailable condition
+// that kubelet adds to initial node status when using GCE.
+// TODO: make upstream kubelet more flexible with overlays and GCE
```

Comment:
It gets cleared by vendor/k8s.io/kubernetes/pkg/controller/route/routecontroller.go which we don't create or run with openshift-sdn because it uses an overlay for the pod traffic.  The routes that get created by the routecontroller are basically "you can reach this Pod CIDR (eg, HostSubnet) via this node" and since our data plane is in the overlay, we're effectively doing that ourselves in OVS by sending packets for a specific HostSubnet to a VXLAN tunnel peer instead of letting that happen via GCE.